config-templates/redhat/odf-local/4.10/custom-parameters.json

```diff
@@ -17,7 +17,8 @@
     "name": "osd-device-path",
     "place-holder": "/dev/sdc,/dev/sde",
     "required": "false",
-    "mutable": "true"
+    "mutable": "true",
+    "regex": "^([a-zA-Z0-9]*[a-zA-Z0-9-_.,]*[a-zA-Z0-9])$"
```

Comment:
This is not correct regex for device paths.
Device path contains `/` as well. For example, `/dev/sda`